aws/resource_aws_autoscaling_group.go

```diff
@@ -204,6 +204,12 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 				},
 			},
 
+			"capacity_rebalance": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
No need for `Default: false` (that's the default).

Reply:
Resolved in cf0e240